desktop/assets/auctions.coffee

```diff
@@ -6,10 +6,12 @@ routes =
   /auctions/reminders
   ''': require('../apps/auctions/client/index.coffee').init
 
+  # TODO: refactor to check paths in order.
+  # Discussion here: https://github.com/artsy/force/pull/1126
   '''
   /sale/.*
-  /auction/.*
-  ''': require('../apps/auction/client/index.coffee').init
+  ^/auction/[^/]+/?$
+  ''': -> require('../apps/auction2/client/index.js').default
```

Comment:
literally just fixed that exact thing for professional buyer's asset package! https://github.com/artsy/force/pull/1172/files#diff-7c9d6756206ba48b2a2cb3293ea995e5L19